source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -152,21 +366,138 @@ provision the following networking API entities, represented in yaml:
     cidr: "bef0:1234:a890:5678::/64"
     enable_dhcp: yes
     gateway_ip: bef0:1234:a890:5678::1
+    ipv6_address_mode: dhcpv6_stateful
   register: ipv6_subnet
 
-- name: create a logical switch port
-  os_port:
+- name: create a router to send RA messages
+  os_router:
     cloud: "{{ cloud_name }}"
     state: present
-    name: lport1
-    network: "{{ ipv6_network.id }}"
-    fixed_ips:
-      - subnet_id: "{{ ipv6_subnet.id }}"
+    name: router0
+    interfaces:
+      - "{{ ipv6_subnet.id }}"
 ~~~~~
 
+### Configuring stateless DHCPv6 on the subnet
+
+To configure stateless dhcpv6 on the subnet, the user should provision
+the following yaml based ansible playbook snippet:
+
+~~~~~
+- name: create a network
+  os_network:
+    cloud: "{{ cloud_name }}"
+    state: present
+    name: netv6
+  register: ipv6_network
+
+- name: attach a subnet to the logical network
+  os_subnet:
+    cloud: "{{ cloud_name }}"
+    state: present
+    network_name: netv6
+    name: subnet_ipv6
+    ip_version: 6
+    cidr: bef0:1234:a890:5678::/64
+    gateway_ip: bef0:1234:a890:5678::1
+    ipv6_address_mode: dhcpv6_stateless
+  register: dhcpv6_subnet
+
+- name: create a router to send RA messages
+  os_router:
+    cloud: "{{ cloud_name }}"
+    state: present
+    name: router0
+    interfaces:
+      - "{{ dhcpv6_subnet.id }}"
+~~~~~
+
+## UI considerations
+
+The subnet ipv6_address_mode parameter should be added to the UI, since
+updates to existing subnets are not possible through the UI, and updates to the
```

Comment:
I think it would be not useful to add ipv6_address_mode to the UI, because Engine has to use dhcp-stateless, because Engine has no UI for fixed IPs.

Reply:
I am missing something on your logic; fixed lengths would only be required for prefix lengths != 64.

meaning that for the 'standard' (I'm calling it standard since it's the only prefix lengths valid for stateless) prefix length it wouldn't be required.

Having said that, if possible, I'd avoid changing the UI.

Reply:
> fixed lengths would only be required for prefix lengths != 64.

What means `fixed lenght` her?

I understood that only a prefix length of 64 would be valid in UI, which seems to be accaptable.

Reply:
Oh, you're right - I meant 'fixed ips' :) 

As in 'fixed IPs would only be required for prefix lengths != 64. 

Ipv6 is making me dyslexic.

Reply:
What is the summary of the discussion? That the UI should accept only 64 bit networks and requires no further change?

Reply:
Updated this section indicating that the only accepted network prefix length is 64.

Since that limitation only applies to the UI, it should be implemented in the UI.

Reply:
If the code changes related to this feature page are not expected to include the ipv6_address_mode support of Engine, please do not write
>  ipv6_address_mode parameter should be added to the UI
If you want to mention this, you could add a section for follow-up features, or just not mention the ipv6_address_mode in the UI section.